index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML TECHNICAL DOCUMENTATION</title>
+</head>
+<body>
+ <h1>HTML Technical DOCUMENTATION</h1> 
+ <h2>Introduction</h2>
+ <p>HTML is an acronym for HyperText Markup Language, and is the standard language for creting web-pages. Every web page you surf on your browser is built using HTML. The primary use of HTML is to provide a proper structure to the web page so all the content or data of the page could represent adequately. A stand-alone HTML can create only static and skeleton looking black and white pages, but with the help of CSS and Javascript,we can create a more interactive and intuitive web page. When we try to visit a website or click on the link, we basically request the server to show us the page, then the server acts on our request and sends us the appropriate HTML document as a response. Then this HTML document parsed by browse, and we are able to see the content.</p>
+  <h3>HTML Editors</h3>
```

Comment:
## Missing Code Block

As per the screenshot, you are missing a code block for a code snippet
Kindly Complete the code Block that is remaining

you can use

`<code> </code>`
or
`<pre> </pre>`